pkg/util/stdio/arm_dup.go

```diff
@@ -7,5 +7,5 @@ import "syscall"
 // DupTo duplicates old fd into the new fd
 // see dup2 and dup3 system calls
 func DupTo(oldfd, newfd int) error {
-	return syscall.Dup3(oldfd, newfd, 0)
+	return syscall.Dup2(oldfd, newfd)
```

Comment:
Why are we switching back here? I do not recall all the context of the original implementation of the call here.

Reply:
just that Dup3 is not supported on all platforms, i.e. windows

Reply:
@alexadhy I see. In this case, can you check that the if we update a visor the output of the update still gets copied over correctly and displayed correctly in the terminal on Linux or Mac? I remember there was a reason we had to switch initially.